_data/changelogs/component-accordion.yml

```diff
@@ -2,6 +2,14 @@ title: Accordion
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Adjusted high contrast styles to ensure visibility and increase consistency.
```

Comment:
Sometimes we refer to this as "high contrast" and other times as "forced colors". Do we have a preference? Are there times (like this) where we would want to use both terms?

Reply:
Resolved to use both terms